pkg/standalone/list.go

```diff
@@ -41,6 +41,11 @@ type runData struct {
 	appCmd     string
 }
 
+func (s *Client) List() ([]ListOutput, error) {
+	// TODO remove this method and have only one List method with a Client receiver
+	return List()
```

Comment:
For now not changing the List method. PR only changes List() method in places that have been tested completely.